pkg/ibmcsidriver/controller_helper.go

```diff
@@ -205,6 +222,14 @@ func getVolumeParameters(logger *zap.Logger, req *csi.CreateVolumeRequest, confi
 		volume.VPCVolume.VolumeEncryptionKey = nil
 	}
 
+	// Add initialOnwer if UID/GID is given as parameter.
+	// Default will be set to 0 i.e root which even if not set will be defaulted to 0 by the VPC RIAAS
+	logger.Info("Adding initial owner...", zap.Any("uid", uid), zap.Any("gid", gid))
```

Comment:
Can we add an if condition to do this only if uid/gid are mentioned
like,
```
if uid != 0 || gid != 0 {
  logger.Info("Adding initial owner...", zap.Any("uid", uid), zap.Any("gid", gid))
  volume.InitialOwner = &provider.InitialOwner{
		GroupID: int64(gid),
		UserID:  int64(uid),
	}
}
```

Am saying this, because the logging info "Adding initial owner" seems to indicate that the user has mentioned the initial owner value in the sc.